spark/hooks/run

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+exec 2>&1
+
+export JAVA_HOME="{{ pkgPathFor "core/jre8" }}"
+export SPARK_CONF_DIR="{{pkg.svc_config_path}}"
+export SPARK_LOG_DIR="{{pkg.svc_var_path}}/logs"
+export SPARK_WORKER_DIR="{{pkg.svc_var_path}}/work"
+export SPARK_NO_DAEMONIZE=1
+
+{{~#if svc.me.leader}}
+exec {{pkg.path}}/sbin/start-master.sh
```

Comment:
Since `sbin` is in your `pkg_bin_dirs`, you can just use `start_master.sh` here without `{{pkg.path}}/sbin/`.